app/presenters/activitypub/activity_presenter.rb

```diff
@@ -27,6 +27,18 @@ def from_status(status)
       end
     end
 
+    def from_status_edit(status_edit)
+      new.tap do |presenter|
+        presenter.id = ActivityPub::TagManager.instance.activity_uri_for(status_edit)
+        presenter.type = 'Update'
+        presenter.actor = ActivityPub::TagManager.instance.uri_for(status_edit.status.account)
+        presenter.published = status_edit.created_at
+        presenter.to = ActivityPub::TagManager.instance.to(status_edit.status)
+        presenter.cc = ActivityPub::TagManager.instance.cc(status_edit.status)
+        presenter.virtual_object = status_edit.status.proper
```

Comment:
This mean the serialized edit is just an `Update` with the contents of the latest version of the status. This doesn't sit right with me.